tests/unit/multi-currency/test-class-frontend-currencies.php

```diff
@@ -33,10 +33,10 @@ class WCPay_Multi_Currency_Frontend_Currencies_Tests extends WP_UnitTestCase {
 	public function setUp() {
 		parent::setUp();
 
+		$this->mock_locale         = $this->createMock( WCPay\MultiCurrency\Locale::class );
```

Comment:
Nitpick: How do you feel about importing the namespace here by adding a `use WCPay\MultiCurrency\Locale;` statement at the top of the file and then using `Locale::class` here rather than the full namespace, and doing the same with the other imports we use in this file? :)

Reply:
I like it, it makes it much easier to read. Updated in https://github.com/Automattic/woocommerce-payments/pull/2408/commits/e1fd9e672cf145dd3cd6524a70c037d06b9d46a6